clusterctl/clusterdeployer/clusterclient.go

```diff
@@ -118,11 +171,15 @@ func (c *clusterClient) GetClusterObjects() ([]*clusterv1.Cluster, error) {
 	return clusters, nil
 }
 
-func (c *clusterClient) GetMachineDeploymentObjects() ([]*clusterv1.MachineDeployment, error) {
+func (c *clusterClient) GetClusterObjects() ([]*clusterv1.Cluster, error) {
```

Comment:
Preserving these methods for other users of the cluster client.
Removing these will be a breaking change.